python/modules/flow_control.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/python
+
+import os, logging
+import modules.file_managment as FM
+
+logger = logging.getLogger("flow control")
+logger.info("Executing flow_control module.")
+
+
+def compare_args(arg1, arg2, comparison):
+    if comparison == "equal" or comparison == "==":
+        if arg1 == arg2:
+            return True
+    elif comparison == "different" or comparison == "!=":
+        if arg1 != arg2:
+            return True
+    elif (comparison == "greater" or comparison == ">" or 
+            comparison == "less" or comparison == "<"):
+        if _greater_or_less(arg1, arg2, comparison):
+            return True
+    else:
+        logger.error("Unknown for TASK_IF comparison type: %s", comparison)
+    return False
+
+def _greater_or_less(arg1, arg2, comparison):
+    try:
+        arg1 = int(arg1)
+        arg2 = int(arg2)
+    except:
+        logger.error("Cannot compare if given parameter is greater/less than "
+        "another when given arguments are not integers: %s, %s", arg1, arg2)
+        return False
+    if comparison == "greater" or comparison == ">":
+        if arg1 > arg2:
+            return True
+    else:
+        if arg1 < arg2:
+            return True
+    return False
+
+def create_ele_list(input_path, wells_params, used_value):   
+    """
+    Arguments:
+    - input_path - path to directories
+    - wells_params - list of base parameters of chosen wells set
+    - used_value - value defying if given dir name reflects well tag or id
+    Returns
+    - verified_dirst- list of dirs which parameters are present 
+                        in wells_params list
+    """
+    all_dir_list = FM.dir_get_names(input_path)
+    verified_dirs = []
+    for well in wells_params:
+        if used_value == "tag":
+            if any(dir_name == well["mp_tag"] for dir_name in all_dir_list):
+                verified_dirs.append(well)
+        elif used_value == "id":
+            if any(dir_name == well["mp_id"] for dir_name in all_dir_list):
+                verified_dirs.append(well)
+        else:
+            logger.error("Unexpected used_value: %s", used_value)
+    return verified_dirs
+
+def get_active_wells(mp_dict, exp_part):
+    """
+    Returns list of active wells from mp_dict for given experiment part.
+    """
+    wells = mp_dict.keys()
+    active_wells = []
+    for well in wells:
+        if mp_dict[well]["exp_part"] == exp_part:
+            active_wells.append(well)
+    return active_wells
+
+def get_wells_base_params(mp_dict, wells, prefix, sufix, exp_part):
```

Comment:
@stork119 if you create an issue correlated with your code review comment you should link them to each other to avoid mess #108
We need names with sufixes and prefixes at this point (parallelize_mp).

> 1. I changed names of mp_id and mp_tag.

mp_id and mp_tag -> wellname_id and wellname_tag
> 2. I didn't change functions according to our conversation:
>     - Using names as id if id is missing is already implemented on read_map_plate level
>     - We want to keep the chance to check out both id or tag depends on user preferation (i.e. to allow swaping them during downloading >=2 part experiments.